File: C/lista1 - estrutura de decisao/exercicio_1.c
#include <conio.h>
#include <stdio.h>

int main() {
	float height, imc, weight;
	
	printf("Informe uma altura: ");
	scanf("%f", &height);
	printf("Informe um peso: ");
	scanf("%f", &weight);
	
	imc = weight / (height * height);
	
	if (imc < 18.5) {
		printf("Abaixo do peso.");
	} else if (imc >= 18.5 && imc < 25) {
		printf("Peso normal.");
	} else if (imc >= 25 && imc < 30) {
		printf("Acima do peso.");
	} else if (imc >= 30) {
		printf("Obeso.");
	}
	
	getch();
	return 0;
}

File: Python/lista1 - estrutura de decisao/exercicio_1.py
height = float(input('Informe uma altura: '))
weight = float(input('Informe um peso: '))

imc = weight / (height * height)

if imc < 18.5:
    print('Abaixo do peso.')
elif 18.5 >= imc < 25:
    print('Peso normal.')
elif 25 >= imc < 30:
    print('Acima do peso.')
elif imc >= 30:
    print('Obeso.')